src/main/scala/scatan/model/ApplicationModel.scala

```diff
@@ -0,0 +1,5 @@
+package scatan.model
+
+import scatan.mvc.lib.Model
+
+final case class ApplicationState() extends Model.State
```

Comment:
Consider having consistent names between files and their contents (i.e ApplicationModel.scala and ApplicationState)

```suggestion
final case class ApplicationModel() extends Model.State
```